### DISCUSSION TITLE
Not working Well: Blur Image

### DISCUSSION THREAD

**Discussion by PowerHazzard** [hacksider/Deep-Live-Cam#168] [Q&A] [answered]
Thanks to this amazing project!
I tried to run this project and I configured the project environments successfully.
But it does not work well when I change the face in the image. When I use CPU, it changes with black square, when I use GPU, it changes the blur image not clear. 
There may be some reasons.
My device environment is like this.
GPU: 8G RAM: 32G
Please help me.
![image](https://github.com/user-attachments/assets/e985a363-c52a-41ff-aa64-7bcc1f4ef2ae)

**Comment by ZeroCool22** [comment]
Ah, but this not replace hair, no point then.

Thx for show it.

**Comment by William-island** [comment]
Same problem. When using CPU, it just replaces the face with black square and works bad with GPU.

**Reply by William-island** [reply, marked as answer]
hi, i have solved the problem. Just strictly follow the official tutorial:
pip uninstall onnxruntime onnxruntime-gpu
pip install onnxruntime-gpu==1.16.3

**Reply by PowerHazzard** [reply]
Thanks. I solved it with your help.
But there is a question.
I had installed onnxruntime-gpu==1.18.0 so this would be great, but it didn't.
What's the reason?
Is this project only related with v1.16.3?

**Comment by xx7Ahmed7xx** [comment]
There is no version 1.16.3 !!

**Reply by StevenCanDrink** [reply]
> There is no version 1.16.3 !!

using python 3.10

**Comment by actorrussell8888-sys** [comment]
I'm having this same issue using RTX5090. Please help me out guys
